src/clientversion.h

```diff
@@ -39,7 +39,7 @@
 #define DO_STRINGIZE(X) #X
 
 //! Copyright string used in Windows .rc files
-#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers and The Zcash developers"
+#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " " COPYRIGHT_HOLDERS_FINAL
```

Comment:
This appears not to be correct. It replaces "The Bitcoin Core Developers and The Zcash developers" with just "The Zcash developers".

Reply:
We made that change ourselves when we forked the codebase. AFAICT the upstream PR fixes this the way they want: the final commit appends a "The Bitcoin Core Developers" line if the main copyright line is changed to not contain it.

Reply:
Yes, but see https://github.com/zcash/zcash/pull/6812/files#r1441169298 .

Reply:
As I said, this is doing precisely what the Bitcoin Core developers wanted. If you think there's a mistake there, it is a mistake in upstream.

Reply:
Yes I think they are incorrect. But more fundamentally, Bitcoin Core doesn't get to change our copyright attributions. Even if the effect of their changes is to incorrectly remove the attribution to them. We have a legal obligation to retain that attribution.

Reply:
```suggestion
#define COPYRIGHT_STR "2015-" STRINGIZE(COPYRIGHT_YEAR) " " COPYRIGHT_HOLDERS_FINAL "; 2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers"
```